awx/main/migrations/0073_v350_schedule_analytics_template.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2016 Ansible, Inc.
+# All Rights Reserved.
+
+from __future__ import unicode_literals
+import random
+import awx.main.fields
+
+from django.db import migrations, models
+import django.db.models.deletion
+from django.conf import settings
+from django.utils.timezone import now, timedelta
+
+import jsonfield.fields
+import jsonbfield.fields
+import taggit.managers
+
+
+def create_system_job_templates(apps, schema_editor):
+    '''
+    Create default system job templates if not present. Create default schedules
+    only if new system job templates were created (i.e. new database).
+    '''
+
+    SystemJobTemplate = apps.get_model('main', 'SystemJobTemplate')
+    Schedule = apps.get_model('main', 'Schedule')
+    ContentType = apps.get_model('contenttypes', 'ContentType')
+    sjt_ct = ContentType.objects.get_for_model(SystemJobTemplate)
+    now_dt = now()
+    random_time = now() + timedelta(minutes=random.randint(-30,30))
+    random_schedule_time = random_time.strftime('%Y%m%dT%H%M%SZ')
+
+    sjt, created = SystemJobTemplate.objects.get_or_create(
+        job_type='gather_analytics --ship',
+        defaults=dict(
+            name='Automation Insights Collection',
+            description='Collect analytics data and send it to Automation Insights',
+            created=now_dt,
+            modified=now_dt,
+            polymorphic_ctype=sjt_ct,
+        ),
+    )
+    if created:
+        sched = Schedule(
+            name='Gather Automation Insights',
+            rrule='DTSTART:%s RRULE:FREQ=DAILY;INTERVAL=1;COUNT=1' % random_schedule_time,
+            description='Automatically Generated Schedule',
```

Comment:
Make this better ... maybe `This schedule was generated by Tower during the install. Will periodically collect system statistical information and send it to Automation Insights. You can view this data via <url>`